EpikCord/__init__.py

```diff
@@ -696,6 +699,11 @@ async def wrapper():
                 await self.heartbeat(True)
 
             elif event["op"] == GatewayOpcode.HEARTBEAT_ACK:
+                heartbeat_ack_time = time_ns()
+                self.discord_latency:int = heartbeat_ack_time - self.heartbeat_time
+                if len(self.latencies) > 10:
+                    self.latencies.pop(0) # pop the first latency
```

Comment:
Popping the first element require rebuilding the list, thus being `O(n)` time complexity, maybe could be by replacing the item `counter % size`